### PR TITLE
Update LegitimateURLShortener.txt

### PR DESCRIPTION
**Pull request by shenzhiming88**
I don't know if this should be added.

### CODE REVIEW

File: LegitimateURLShortener.txt [hunk 1]
```diff
@@ -1950,6 +1950,9 @@ $removeparam=fromRankId
 ||bilibili.com^$removeparam=refer_from
 ! https://show.bilibili.com/platform/home.html?msource=pc_web
 ||bilibili.com^$removeparam=msource
+! https://github.com/the1812/Bilibili-Evolved/issues/2247
+! This script is used to add the spm_id_from parameter to the hyperlink on the web page.
+||s1.hdslb.com/bfs/seed/jinkela/short/auto-append-spmid.js
```
Comment (iam-py-test) [inline LegitimateURLShortener.txt:1955]: I do not think this is really the purpose of Legit URL Shortener, but I'd have to ask Imre.
@DandelionSprout your opinion?

**Comment by iam-py-test** [comment]
I keep forgetting that adding a commit somewhere else with the word _Fix_ closes this

**Comment by iam-py-test** [comment]
@shenzhiming88 merge conflicts 
@DandelionSprout opinion?

**Comment by iam-py-test** [comment]
@DandelionSprout?

**Comment by shenzhiming88** [comment]
> @DandelionSprout?

Maybe he is too busy to see this.

**Comment by iam-py-test** [comment]
> 
> 
> > @DandelionSprout?
> 
> Maybe he is too busy to see this.

Probably. However, I did notice he [committed something](https://github.com/DandelionSprout/adfilt/commit/229ab4de3f50c430286c8db16150fedd0accf3e7), and the longer this PR remains unmerged, the more likely there are to be conflicts

**Comment by DandelionSprout** [comment]
I've been bingeplaying Sims 3 for pretty much the entire 3 days straight, so give me a few minutes to look into this.

File: LegitimateURLShortener.txt (hunk 1 above)
Comment (DandelionSprout) [reply LegitimateURLShortener.txt:1966]: After thinking about it a bit, I think there are grounds to shorten it to simply `/auto-append-spmid.js` even.

**Pull request merged**